tests/cucumber/steps/steps.js

```diff
@@ -4454,6 +4454,33 @@ module.exports = function getSteps(options) {
     }
   );
 
+  Given(
+    'a base64 encoded program bytes for heuristic sanity check {string}',
+    async function (programByteStr) {
+      this.seeminglyProgram = new Uint8Array(
+        Buffer.from(programByteStr, 'base64')
+      );
+    }
+  );
+
+  When('I start heuristic sanity check over the bytes', async function () {
+    this.actualErrMsg = undefined;
+    try {
+      new algosdk.LogicSigAccount(this.seeminglyProgram); // eslint-disable-line
+    } catch (e) {
+      this.actualErrMsg = e.message;
+    }
+  });
+
+  Then(
+    'if the heuristic sanity check throws an error, the error contains {string}',
+    async function (errMsg) {
+      if (errMsg !== '')
+        assert.strictEqual(this.actualErrMsg.includes(errMsg), true);
+      else assert.strictEqual(this.actualErrMsg === undefined, true);
```

Comment:
nit: you can use `assert.ok(this.actualErrMsg.includes(errMsg))` and `assert.strictEqual(this.actualErrMsg, undefined)`, which are a little more concise

Reply:
good point